### ISSUE TITLE
Deprecated API overloads are misnamed

### ISSUE THREAD

**Issue by Mordil** [airbnb/HorizonCalendar#159]
It seems that with the item model provider API renames, the deprecated overloads used the new name without the `model` part - causing builds to fail with 1.13.

```swift
.withMonthHeaderItemModelProvider { [unowned self] in
 ^^^^^ Value of type 'CalendarViewContent' has no member 'withMonthHeaderItemModelProvider'
```

In `Deprecations.swift` I see [`withMonthHeaderItemProvider`](https://github.com/airbnb/HorizonCalendar/blob/master/Sources/Public/Deprecations.swift#L175)

**Comment by bryankeller** [comment]
Oh shoot, I see the mistake I made. So sorry for this @Mordil + anyone else who started running into this broken behavior 😔 Fix will be in 1.13.1 in a few min

**Comment by bryankeller** [comment]
Fixed in 1.13.1 - sorry again for the trouble!

**Comment by bryankeller** [comment]
Just no realizing that 1.*13* was the broken build. Love the irony 😂
